public/externalLibs/inspector/inspector.js

```diff
@@ -1,225 +1,4 @@
 (function (exports) {
-  var container = document.createElement('div');
```

Comment:
The functions I've removed don't seem to be referenced anymore. This means that this file (named inspector.js) is now only responsible for highlighting lines (when debugging).